root/static/scripts/common/utility/privileges.js

```diff
@@ -109,6 +109,13 @@ export function isEditingEnabled(editor: EditorPropT): boolean {
   return (editor.privileges & EDITING_DISABLED_FLAG) === 0;
 }
 
+export function isVotingEnabled(editor: EditorPropT): boolean {
+  if (editor == null) {
+    return false;
+  }
+  return !isBot(editor) && isEditingEnabled(editor);
+}
```

Comment:
Same question as above; I'm afraid it might be unclear what the difference between `isVotingEnabled` and `editorMayVote` is, and we might end up calling the former where we should really only be using the latter. Maybe they should just stay merged? (I also noticed that `is_voting_enabled` on the server checks for a confirmed email address but the JS version doesn't.)

Reply:
Yeah. This page only expects privileges in its "editor", which is why I made it check the date elsewhere. I guess I could also expect a date here though and have it all here.